bot/exts/evergreen/status_cats.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from http import HTTPStatus
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
```

Comment:
Is this needed? There doesn't seem to be any logging within this cog.

Reply:
Did that out of habit. Thanks for noticing!